modules/phase_field/src/ics/MultiSmoothCircleIC.C

```diff
@@ -92,7 +92,7 @@ MultiSmoothCircleIC::computeCircleCenters()
     //Vary circle center positions
     unsigned int num_tries = 0;
 
-    Real rr = 0.0;
+    Real rr = -1.0;
```

Comment:
Why? If you want the `if` clause to be true for `_bubspac == 0` then change the `<` to a `<=`. Putting a -1 here looks abitrary.